public/index.html

```diff
@@ -8,6 +8,8 @@
     <meta
       name="description"
       content="Web site created using create-react-app"
+      http-equiv="Content-Security-Policy"
```

Comment:
when I was testing the code, I tried to deply to gh-pages and the API didn't work. I googled and realised that its because of some http issue, not being https. I then went to add in these 2 lines into the meta of html page. Only after that I realised the API call I forgot to put https in the url. It worked after that, and I forgot to remove these 2 lines in html page. I don't know whether if I remove these 2 lines, the API call will still work on gh-pages or not.